### PR TITLE
Adds new key-value pair for CAPV jobs

### PR DESCRIPTION
**Pull request by srm09**
This PR adds a new secret ref to read the thumbprint for the vCenter instanced used for CAPV CI.

### CODE REVIEW

**Comment by srm09** [comment]
/approve
waiting for an `lgtm` on this one.

**Comment by srm09** [comment]
@chaodaiG Could you take a look at the PR and approve it at your convenience?

**Comment by chaodaiG** [comment]
/assign figo akutz
these are the OWNERS listed for https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/OWNERS

**Comment by srm09** [comment]
image
@chaodaiG Assigned you since the comment says that I need an approval from an approver in each one of the files above. Can the PR be merged even without the second approval, in which case I can ping others

**Comment by chaodaiG** [comment]
> @chaodaiG Assigned you since the comment says that I need an approval from an approver in each one of the files above. Can the PR be merged even without the second approval, in which case I can ping others

Agree that it will be faster for me to approve this PR, but unfortunately I don't have enough context on `config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-presubmits.yaml`, it would be the best to get approval from approver of this directory first as they have most context

**Comment by srm09** [comment]
Both @lubronzhan and I are [approvers](https://github.com/kubernetes/test-infra/blob/1971640e4b553210812b4b63a6f1a19d5a1b7971/config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/OWNERS#L11-L12)  for that directory.

**Review by chaodaiG — APPROVED** [review]

**Pull request merged**
